packages/babel-preset-env/data/built-ins.json

```diff
@@ -1,4 +1,204 @@
 {
+  "es6.array.every": {
+    "chrome": "5",
+    "opera": "10.1",
+    "edge": "12",
+    "firefox": "2",
+    "safari": "3.1",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.array.filter": {
+    "chrome": "5",
+    "opera": "10.1",
+    "edge": "12",
+    "firefox": "2",
+    "safari": "3.1",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.array.for-each": {
+    "chrome": "5",
+    "opera": "10.1",
+    "edge": "12",
+    "firefox": "2",
+    "safari": "3.1",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.array.index-of": {
+    "chrome": "5",
+    "opera": "10.1",
+    "edge": "12",
+    "firefox": "2",
+    "safari": "3.1",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.array.is-array": {
+    "chrome": "5",
+    "opera": "10.5",
+    "edge": "12",
+    "firefox": "4",
+    "safari": "4",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.array.last-index-of": {
+    "chrome": "5",
+    "opera": "10.1",
+    "edge": "12",
+    "firefox": "2",
+    "safari": "3.1",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.array.map": {
+    "chrome": "5",
+    "opera": "10.1",
+    "edge": "12",
+    "firefox": "2",
+    "safari": "3.1",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.array.reduce": {
+    "chrome": "5",
+    "opera": "10.5",
+    "edge": "12",
+    "firefox": "3",
+    "safari": "4",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.array.reduce-right": {
+    "chrome": "5",
+    "opera": "10.5",
+    "edge": "12",
+    "firefox": "3",
+    "safari": "4",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.array.some": {
+    "chrome": "5",
+    "opera": "10.1",
+    "edge": "12",
+    "firefox": "2",
+    "safari": "3.1",
+    "node": "0.1",
+    "ie": "9",
+    "android": "4",
+    "ios": "6",
+    "electron": "1.1"
+  },
+  "es6.array.sort": {
+    "chrome": "63",
+    "opera": "50",
+    "edge": "12",
+    "firefox": "5",
+    "safari": "10.1",
```

Comment:
This is a bit annoying... according to compat-table, Safari actually _regressed_ on this, and Safari 11 has the same problem as Chrome 62 :cry:

Reply:
Maybe it's worth to support ranges there?